numba_dpex/core/parfors/reduction_helper.py

```diff
@@ -399,7 +399,7 @@ def copy_final_sum_to_host(self, parfor_kernel):
 
         # Create a local variable storing a pointer to a DPCTLSyclQueueRef
         # pointer.
-        curr_queue = ir_builder.get_queue(exec_queue=parfor_kernel.queue)
+        queue_ref = ir_builder.get_queue(exec_queue=parfor_kernel.queue)
```

Comment:
Let us stay consistent with naming. `s/ir\_builder/kl\_builder/g`

Reply:
Also, I am assuming the `get_queue` will itself be cleaned up in a follow up PR?

Reply:
Yes, it is not part of builder, so it's gonna be removed. Either by getting queue on llvm level or removing this code out of the class.